common/network-common/pom.xml

```diff
@@ -45,6 +45,22 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.fusesource.leveldbjni</groupId>
```

Comment:
That's sort of orthogonal, though. My code pulls leveldb directly, not transitively through the network libs.